Update LGFX_ESP32S3_RGB_MakerfabsParallelTFTwithTouch70.h

byronin opened the pull request:
Hi! I added setup for 4.3" display.

HarryVienna commented:
Hi,

Thanks for your input!
Can you please create a seperate header file LGFX_ESP32S3_RGB_MakerfabsParallelTFTwithTouch43.h for your display?

Then you just need to include your header file in gui.cpp and adapt the resolution :-)

byronin commented:
okay I will do.

byronin commented:
Hi, I’ve made the necessary updates for the 4.3" display. I also modified lv_conf.h to reduce screen flickering and tearing. Additionally, I added a simple spinbox demo.

HarryVienna commented:
Where is the include of the 7 inch display gone?

byronin commented:
Sorry, I forgot to add them. They are in rn.

HarryVienna commented:
Where?

byronin commented:
https://github.com/byronin/Makerfabs-Sunton-ESP32-S3-7-Display-with-LovyanGFX-and-LVGL/tree/develop/src/gfx

On Wed, May 7, 2025 at 5:11 PM Harald ***@***.***> wrote:

> *HarryVienna* left a comment
> (HarryVienna/Makerfabs-Sunton-ESP32-S3-7-Display-with-LovyanGFX-and-LVGL#4)
> <https://github.com/HarryVienna/Makerfabs-Sunton-ESP32-S3-7-Display-with-LovyanGFX-and-LVGL/pull/4#issuecomment-2860550828>
>
> Where?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/HarryVienna/Makerfabs-Sunton-ESP32-S3-7-Display-with-LovyanGFX-and-LVGL/pull/4#issuecomment-2860550828>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ALE6HIEFKAIMPDYO3LIIY4L25KAKNAVCNFSM6AAAAAB4UPSMNOVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDQNRQGU2TAOBSHA>
> .
> You are receiving this because you authored the thread.Message ID:
> <HarryVienna/Makerfabs-Sunton-ESP32-S3-7-Display-with-LovyanGFX-and-LVGL/pull/4/c2860550828
> @github.com>
>

HarryVienna commented:
Yes that's your fork
